cmd/maestro/server/routes.go

```diff
@@ -79,9 +79,14 @@ func (s *apiServer) routes() *mux.Router {
 	apiV1ResourceRouter.HandleFunc("/{id}", resourceHandler.Patch).Methods(http.MethodPatch)
 	apiV1ResourceRouter.HandleFunc("/{id}", resourceHandler.Delete).Methods(http.MethodDelete)
 	apiV1ResourceRouter.Use(authMiddleware.AuthenticateAccountJWT)
-
 	apiV1ResourceRouter.Use(authzMiddleware.AuthorizeApi)
 
+	apiV1ResourceBundleRouter := apiV1Router.PathPrefix("/resourcebundles").Subrouter()
```

Comment:
I recommend to separate words with hyphens. It is more user-friendly when it comes to long-path segmented URIs.

Reply:
update to `/resource-bundles`